games/minetest/Portfile

```diff
@@ -44,27 +45,44 @@ post-extract {
     }
 }
 
-license                 LGPL-2.1+
+license                 LGPL
 categories              games
 platforms               darwin
-maintainers             nomaintainer
+maintainers             @Zweihorn
 description             open source infinite-world block sandbox game with survival and crafting
-long_description        ${description}.
-
-depends_build-append    path:bin/doxygen:doxygen \
-                        port:mesa
+long_description        - ${description} - \
+                         \
+                        Minetest (usually abbreviated as MT) is an open source voxel game \
+                        engine. Play one of our many games, mod a game to your liking, make your \
+                        own game, or play on a multiplayer server. \
+                         \
+                        Available for Windows, macOS, GNU/Linux, FreeBSD, OpenBSD, DragonFly \
+                        BSD, and Android. \
+                         \
+                        ** Minetest at <https://www.minetest.net> \
+                        * MT mods at <https://content.minetest.net/> * \
+                        * Database explained at <https://wiki.minetest.net/Database_backends> ** \
+                         \
+                        This ${name} port provides a cutting edge Minetest (stable).
+
+homepage                https://www.minetest.net
+
+depends_build-append    port:doxygen
 
 depends_lib-append      port:irrlichtmt \
-                        path:include/turbojpeg.h:libjpeg-turbo \
+                        port:libjpeg-turbo \
```

Comment:
revert to the original with the path-style dependency

Reply:
Please explain and give further advice.

I do not understand why this should be required.

Over time the CMake was improved upstream by the MT devs in a way that the new addition of an explicit `port:libjpeg-turbo` in the new Portfile finds the needed MacPorts file automatically. Thus the path would be redundant and is removed.

This was tested by me on a clean install.

Reply:
as I said before, please keep all the "path-style" dependencies - you can look them up in the MacPorts guide. 

In short, for some ports there is a "-devel" subport or some other drop-in replacement which can fulfill the dependency requirement as well. To allow that one should use the path-style dependency instead of adding the explicit port dependency.